lib/pusher/client.rb

```diff
@@ -88,6 +98,10 @@ def encrypted?
       @scheme == 'https'
     end
 
+    def cluster=(cluster)
+      @host = "api-#{cluster}.pusher.com"
```

Comment:
I don't understand why we should set `@port=nil` when setting `@cluster`. cluster option only interacts with the host, not with the port. Why should we force the user to configure `@scheme` when configuring `@host` ?